src/ListMenu/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { View } from '../../src'
+
+const ListMenu = ({
+   activeBackgroundColor,
+   activeTextColor,
+   children,
+   buttonBorderRadius,
+   buttonPaddingHorizontal,
+   buttonSize,
+   dividerLineStyle,
+   showActiveArrow,
+   textColor
+}) => {
+   // Clone ListMenuButton children and pass props
+   const listMenuButtons = React.Children.map(children, (child, i) => {
```

Comment:
Because this is a pretty expensive operation, I think it would be worthwhile memoizing it. Should be pretty straightforward, just like memoizing anything else. Your only 2nd arg param to `useMemo` will be children. I'm fairly confident React will only cause children to change if the children actually change (which is more surprising than it sounds).

Reply:
Happy to help if with this if you'd like